app/services/TtpService.scala

```diff
@@ -234,4 +240,21 @@ object TtpService {
     case someAmount: UpfrontPaymentAnswers.DeclaredUpfrontPayment => Some(someAmount.amount)
     case UpfrontPaymentAnswers.NoUpfrontPayment                   => None
   }
+
+  /**
+   * If email matches the one from the eligibility API - ETMP
+   * If new email entered on the screen which doesn't match the ETMP one - TEMP
+   */
+  private def deriveCustomerDetail(journey: Journey.Stages.EmailVerificationComplete): Option[List[CustomerDetail]] = {
+    val emailFromEligibilityResponse: Option[List[Option[String]]] =
```

Comment:
I think we'd need to only check against emails in the eligibility response which have type `emailSource` equal to `ETMP` so we could apply a filter to the `List` in the eligibility response. 

Also, since email addresses are case insensitive, we could `.toLowerCase` the eligibility response emails and the email in our journey before checking for inequality.

Reply:
I think I've implemented how you've asked